datalad/interface/common_opts.py

```diff
@@ -33,6 +33,13 @@
     constraints=EnsureInt() | EnsureNone(),
     doc="""limit recursion into subdataset to the given number of levels""")
 
+shared_access_opt = Parameter(
+    args=('--shared-access',),
```

Comment:
I thought so. But I'd like to avoid another "create_sibling" situation. We had to shorten it, because it was the only one. But now there are two -- but the command name never got longer again. These things can accumulate to an API structure that communicates more "history" than necessary. I'd prefer to first have the features implemented, not just envisioned.